apps/happy-blocks/src/support-page/view.scss

```diff
@@ -0,0 +1,151 @@
+@import "@automattic/typography/styles/variables";
+
+.hb-support-page-embed {
+	border-radius: 4px;
+	padding: 29px 30px;
+
+	display: flex;
+	gap: 16px;
+	flex-direction: column;
+
+	background: #fff;
```

Comment:
Could we establish some css constants/variables file and use those instead, please? Properties such as colors, font families, font-sizes and others.